k210.svd

```diff
@@ -2556,11 +2556,352 @@
+                            <name>dir_search_data_rdy</name>
+                            <description>Sound direction searching data ready interrupt event</description>
+                            <bitRange>[0:0]</bitRange>
+                        </field>
+                        <field>
+                            <name>voc_buf_data_rdy</name>
+                            <description>Voice output stream buffer data ready interrupt event</description>
+                            <bitRange>[1:1]</bitRange>
+                        </field>
+                        <!-- reserved bits 2..31 -->
+                    </fields>
+                </register>
+                <register>
+                    <name>int_mask</name>
+                    <description>Interrupt Mask Register</description>
+                    <addressOffset>0x12c</addressOffset>
+                    <fields>
+                        <field>
+                            <name>dir_search_data_rdy</name>
+                            <description>Sound direction searching data ready interrupt event</description>
+                            <bitRange>[0:0]</bitRange>
+                        </field>
+                        <field>
+                            <name>voc_buf_data_rdy</name>
+                            <description>Voice output stream buffer data ready interrupt event</description>
+                            <bitRange>[1:1]</bitRange>
+                        </field>
+                        <!-- reserved bits 2..31 -->
+                    </fields>
+                </register>
+                <register>
+                    <name>sat_counter</name>
+                    <description>Saturation Counter</description>
+                    <addressOffset>0x130</addressOffset>
+                    <fields>
+                        <field>
+                            <name>counter</name>
+                            <description>Counter</description>
+                            <bitRange>[15:0]</bitRange>
+                        </field>
+                        <field>
+                            <name>total</name>
+                            <description>Total</description>
+                            <bitRange>[31:16]</bitRange>
+                        </field>
+                    </fields>
+                </register>
+                <register>
+                    <name>sat_limits</name>
+                    <description>Saturation Limits</description>
+                    <addressOffset>0x134</addressOffset>
+                    <fields>
+                        <field>
+                            <name>upper</name>
+                            <description>Upper limit</description>
+                            <bitRange>[15:0]</bitRange>
+                        </field>
+                        <field>
+                            <name>lower</name>
+                            <description>Lower limit</description>
```

Comment:
Maybe this one is `bottom`, but I'm not qualified in this area.

Reply:
Same—I don't really know what the saturation limits do here, it's not described in the API PDF, it's also not used in any of the examples.

Reply:
Oh you mean the parameter is called `bottom` in `apu_voc_set_saturation_limit`, sure, can change it to that.

Reply:
Yes, it's called `bottom` in `apu.c` file.
P.S.: It would be better if you fix things in separate commit so I can review changes quickly

Reply:
Renamed in an added commit. I'm used to projects that prefer whole commits, that's why I squashed every time.